packages/salesforcedx-vscode-automation-tests/test/specs/createOasDoc.e2e.ts

```diff
@@ -328,7 +328,7 @@ describe('Create OpenAPI v3 Specifications', () => {
 
           // Get all open tabs
           const openTabs = await editorView.getOpenTabs();
-          if (!openTabs || openTabs.length === 0) {
+          if (openTabs?.length === 0) {
```

Comment:
I think the optionalChain is unnecessary since getOpenTabs says it return an array